include/ipc/ipc_service.h

```diff
@@ -116,6 +116,21 @@ struct ipc_ept_cfg {
 	void *priv;
 };
 
+/** @brief Open an instance
+ *
+ *  Optional function to be used to open an instance before being able to
```

Comment:
I marked this as optional because some backend could decide to `open()` the instance at `init()` time (that is in the `.init` hook function of the device). This is something that I tried to do before discovering that the stack size available at `init` time is too small to accomodate for the `rpmsg_init_vdev`, so I had to migrate to a proper `open()` function.

Reply:
Thanks for the clarification. What about:
- make `ipc_service_open_instance` usage mandatory
- `backend->open_instance`  would be optional depending on backend
- 'ipc_service_open_instance' just do nothing if `backend->open_instance`  is NULL
This offer a "stable" API for user with possibility to implement or not the backend open instance

Reply:
OK